accepted/0017-add-funding-support.md

```diff
@@ -0,0 +1,138 @@
+# Add `funding` support to `package.json`
+
+## Summary
+
+This RFC identifies an initial means of tooling to describe & notify consumers of a package's monetary support information. `npm` has been [vocal about our commitment](https://blog.npmjs.org/post/187382017885/supporting-open-source-maintainers) to providing a means for package maintainers to more sustainably support their work & this reflects an initial step toward that goal. 
+
+## Motivation
+
+Package maintainers want to clearly indentify how their software is currently, or could be in the future, supported monetarily.
+
+## Detailed Explanation
+
+* See [Prior Art](#prior-art)
+* Provide a means to **reference** existing backers or funding opportunities in `package.json`
+* Provide a means to **define the type** of backing or funding opportunities in `package.json`
+* Provide a means to **notify** package consumers of backers or funding opportunites with the cli
+* Provide a means to **view** dependency's backers or funding opportunites with the cli
+* Provide a means to **view** a package's backers or funding opportunites
```

Comment:
The "backers" term should be defined here, or removed.  To my interpretation, this would imply that we're supporting a way to list out the people supporting a package, which feels very out of scope at this stage.